.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -0,0 +1,10 @@
+Please review our [Contribution Guidelines](https://github.com/avinoth/elixir-companies/blob/master/CONTRIBUTING.md) to ensure your changes abides by it.
+
+Make sure your changes ticks off the below checklist.
```

Comment:
Let's tell them what to do:

"Review your changes and complete the checklist below:"